src/local-editor-service.ts

```diff
@@ -140,6 +139,24 @@ export class LocalEditorService extends EditorService {
     };
   }
 
+  _generateAutoCompletionForElement(e: Element): string {
+    let autocompletion = `<${e.tagName}`;
+    let tabindex = 1;
+    if (e.attributes.length) {
+      autocompletion += ` $${tabindex++}`;
```

Comment:
This resolves to e.g. "$1" but I think the correct syntax that editors expect is "${1}".

Because we basically want to generate the template string interpolation syntax, this is probably one of the rare times where template string interpolation doesn't improve legibility.

What about this instead? 

```javascript
" ${" + tabindex++ + "}"
```